FillDB.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
FillDB name sounds bit confusing. We are grabbing all stuff from NULP site but not just filling DB with dummy data :)